packages/components/src/components/video-player/video-player.scss

```diff
@@ -57,14 +65,14 @@
             margin: 0.8rem 0;
             cursor: pointer;
 
-            &:before {
-                content: '';
-                display: block;
-                height: 2rem;
-                width: 100%;
-                position: absolute;
-                bottom: 67%;
-            }
+            // &:before {
```

Comment:
do we need this for future reference or something ?

Reply:
i forgot to delete i think.